### PR TITLE
Add keras-js@0.3.0 w/ git auto-update

### LINKED ISSUE #11364: [Request] Add keras-js
**Library name:** keras-js
**Git repository url:** https://github.com/transcranial/keras-js
**npm package name or url** (if there is one): https://www.npmjs.com/package/keras-js
**License (List them all if it's multiple):** MIT
**Official homepage:** https://transcranial.github.io/keras-js
**Wanna say something? Leave message here:**

### PR DESCRIPTION
**Pull request by dakshshah96**
close #11364, cc @transcranial

Pull request for issue: #11364 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/transcranial/keras-js
 * Official website (optional, not the repository): https://transcranial.github.io/keras-js
 * NPM package url (optional): https://www.npmjs.com/package/keras-js
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 95
   - Count of stars: 2,318
   - Count of forks: 195
 * NPM download stats (optional):
   - Downloads in the last day: 0
   - Downloads in the last week: 26
   - Downloads in the last month: 126

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/keras-js/package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    "machine",
+    "neural",
+    "networks",
+    "javascript",
+    "webgl",
+    "gpu"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/transcranial/keras-js"
```
Comment (extend1994) [inline ajax/libs/keras-js/package.json]: I know it's correct for "rul" but it's different from the upstream. Not sure if we should follow the upstream?

Comment (dakshshah96) [inline ajax/libs/keras-js/package.json]: As told by @PeterDaveHello in another PR, the repository url being valid is enough. It doesn't need to match the upstream in the format it's written. :)

Comment (PeterDaveHello) [inline ajax/libs/keras-js/package.json]: This is fine for GitHub repos, no worry.

Comment (dakshshah96) [inline ajax/libs/keras-js/package.json]: @extend1994 Can you update the review? :)

**Pull request merged**